src/pe_reports/report_generator.py

```diff
@@ -76,9 +93,21 @@ def convert_to(folder, source, timeout=None):
         folder,
         source,
     ]
+
+    # Bandit triggers B603 here, but we're using subprocess.run()
+    # safely here, since the args input variable to subprocess.run is sanitized
+    # and secure. Since that value is hard-coded, the risks mentioned by B603
+    # are minimal For more details on B603 see here:
```

Comment:
Add a missing period.
```suggestion
    # are minimal. For more details on B603 see here:
```